pages/learn/develop/blocks.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Develop with blocks
+excerpt: A drop-in chunk of functionality that can be added to your application to provide.
+---
+
+# Getting Started with blocks
+
+A software package is a collection of individual files or resources that are packed together to provide a certain functionality as part of a larger system. Blocks are just that. They are drop-in, independent chunks of functionality built to handle a specific task, allowing you to focus on solving the hard problems instead.
+
+A block is a container image that can be added to a multi-container app in order to provide specific functionality. Blocks are intended to reduce the friction of application development by making it easier to re-use functionality, much like software packages. Similar to how software developers don't want to rewrite common functions, this is a DRY approach for IoT developers to not create containers of every function and re-use what's already out there for their application to work.
+
+For example, getting a browser to run well in a container is a tedious task. So, why not use the [browser block][browser].
+
+
+<img src="/img/blocks/diagram-1.png" width="100%">
+
+## Develop with a block
+
+Blocks being a container image can be added to a multi-container app. To use a block in your app, add the block's image as a new service in the `services` section of your app's `docker-compose.yml` file. This is similar to how we would [build multicontainer apps][multicontainer].
```

Comment:
"As Blocks are a container image, they can be added to a multi-container Fleet."

Lots of use of app, when I think we want to keep to Fleet, to tie in with the mission and the words the user can see and use in the Dashboard.